docs/developing/integrate/neon_transfer.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Neon Transfer Client
+proofedDate: na
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: Vale not setup for mdx on this branch -- needs merge of branch with fix to do full proof. Grammarly proofing done
+---
+
+import { DemoFrame } from '@site/src/components/DemoFrame';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+This page explains how to pass tokens between Solana and Neon EVM programmatically.
+
+## TL;DR
+
+- [NeonPass UI](https://devnet.neonpass.live) is available as an npm package
+- [React demo](https://codesandbox.io/s/neon-transfer-demo-z93nlj) available.
+
+## Introduction
+
+This page describes how to use the Neon token pass tool: providing the same functionality as the [NeonPass UI](https://devnet.neonpass.live).
+The tool is an npm package that supports the transfer of SPL or ERC-20 tokens between Solana and Neon EVM.
+
+<!-- todo confirm ok to remove only NEON -- we have other supported SPL tokens -->
+
+## Prerequisites
+
+- You need to provide Solana and Neon EVM providers from supported official libraries such as `@solana/web3.js` or `web3.js`
+, `ether.js`, and `WalletConnect`. 
+- Supported browser plugins (e.g. Phantom or MetaMask wallets).
+
+<!-- I don't understand provide providers above -->
```

Comment:
@oable

Reply:
Should provide access for providers in browser plugins (It is developer abstractions)

Reply:
Can you have a go at saying this differently please -- ie use different terms to explain the concept!

Reply:
@oable

Reply:
Perhaps:
"
You need to work with Solana-supported and EVM-supported browser plugins from supported official libraries such as `@solana/web3.js` or `web3.js`, `ether.js` and `WalletConnect`. 
"

Reply:
@oable I note that you have merged this PR. I still have this item that I did not understand. Please can you assist further?